src/login.js

```diff
@@ -90,8 +87,8 @@ async function login() {
 
     const access_token = res.data.access_token;
     if (access_token === undefined) throw 'Login failed, please try again. LumiNUS sucks sometimes.';
-    if (PRINT) console.log('done!');
+    if (readPrint()) console.log('done!');
     return 'Bearer ' + access_token;
 }
 
-module.exports = { login };
+module.exports = { login, getUsername };
```

Comment:
Any reason for exporting `getUsername` as well?

Reply:
Hmm, that can be safely deleted. It was for running the tests